ansible/deploy_db.yml

```diff
@@ -99,3 +99,43 @@
     - celery
   roles:
     - {role: java, tags: java}
+
+# Disables transparent huge pages (THP)
+- name: Disable THP
+  become: true
+  hosts:
+    - postgresql
+    - couchdb
+    - redis
+    - elasticsearch
+    - rabbitmq
+    - zookeeper
+    - riakcs
+    - formplayer
+  tags: 'kernel_tuning'
+  tasks:
+    - stat:
+        path: /sys/kernel/mm/transparent_hugepage/enabled
+      register: thp_enabled
+    - stat:
+        path: /sys/kernel/mm/transparent_hugepage/defrag
+      register: thp_defrag
+    - name: Sets THP enabled to never
+      raw: echo never > /sys/kernel/mm/transparent_hugepage/enabled
+      when: thp_enabled.stat.exists
+    - name: Sets THP defrag to never
+      raw: echo never > /sys/kernel/mm/transparent_hugepage/defrag
+      when: thp_defrag.stat.exists
+    - name: Adds transparent_hugepage=never to default grub config
```

Comment:
If this causes an error in the grub config that causes us to not be able to boot, what would be the best way to troubleshoot it?